docs/source/query.rst

```diff
@@ -87,7 +98,7 @@ Nested Keys
 To match **nested** keys, avoid nesting the filter arguments, but instead use the ``.``-operator.
 For example, if the documents shown in the example above were all nested like this:
 
-.. code-block:: python
+.. code-block:: bash
```

Comment:
Same comment here. If you want to use bash syntax you need to add quotations around the JSON objects.